CHANGELOG.md

```diff
@@ -3,13 +3,84 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.7.0
+
+_Mar 24, 2022_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- ✏ Add a new editing API with better support for server-side persistence and validation (#3963) @m4theushw
+
+  The new API is only accessible by enabling the following flag:
+
+  ```tsx
+  <DataGrid experimentalFeatures={{ newEditingApi: true }} />
+  ```
+
+  Users relaying on the old API (legacy) don't need worry as it will continue to work until v6.
```

Comment:
```suggestion
  Users relying on the old API (legacy) don't need to worry as it will continue to work until v6.
```

Btw @m4theushw, I think it'd make sense to officially deprecate the old API.